apps/jobs/views.py

```diff
@@ -318,16 +378,70 @@ def create(self, request, *args, **kwargs):
                         status=status.HTTP_406_NOT_ACCEPTABLE,
                     )
 
-        ## Save the data into the database
-        # Update the fields
-        serializer = self.get_serializer(data=request.data)
-        serializer.is_valid(raise_exception=True)
-        self.perform_create(serializer)
-        data = serializer.validated_data
-        headers = self.get_success_headers(data)
-        return Response(
-            serializer.data, status=status.HTTP_201_CREATED, headers=headers
-        )
+        # Check if the given user_id actually belongs to the user
+        # This check is necessary because only specific users can update their profile
+
+        # get the user_id from the url path
+        if user_id := search("\w{32}", request.stream.path):
+            user_id = user_id[0]
```

Comment:
please follow security checks

Reply:
Removed the check for user_id comparison, now only the user_id present in access-token will be used in the update method, especially for updating user_information